components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -124,6 +124,7 @@ abstract class EngineSession(
             fileName: String? = null,
             contentLength: Long? = null,
             contentType: String? = null,
+            path: String? = null,
```

Comment:
nit: Maybe `defaultDownloadPath`. Wdyt?

Reply:
`Environment.DIRECTORY_DOWNLOADS` could be the default value something like `path: String =Environment.DIRECTORY_DOWNLOADS `